README.md

```diff
@@ -20,12 +20,18 @@ Planned features in no particular order:
 - More administration features
 - Update configs
 
+You might also find useful information in the [Online Documentation](https://github.com/jlandersen/vscode-kafka/blob/master/docs/README.md)
+or you can read this documentation inside your editor (can run in vscode, vscodium, che, theia...) with the command `Open Documentation` available with `Ctrl+Shift+P`:
```

Comment:
remove `(can run in vscode, vscodium, che, theia...)`, it was meant as a review  comment  ;-)

Reply:
fixed